.gitattributes

```diff
@@ -1,3 +1,7 @@
+# Always use Unix line endings for Unix shell scripts.
+gradlew		text eol=lf
+*.sh		text eol=lf
```

Comment:
Would be nice to put `text eol=lf` in the same column as in the entries below.

Reply:
Done. Note that the file is using tabs and thus subject to your tab expansion setting. But in `vim` the columns show up aligned now.